strongLiveVariableAnalysis/SLVA.cpp

```diff
@@ -105,23 +118,26 @@ class LiveAnalysis {
 			}
 			auto FI = BB->begin();
 			const Instruction *FInsn = &(*FI);
```

Comment:
Can't we just change `FInsn` to `FInsn = BB->getTerminator()`. That seems like a minimalistic change to fix convergence else we may have to make worklist of Instructions instead of BasicBlocks. 

Ideally, we should be checking for convergence at the end of every Instruction.